Source/Core/Core/Core.cpp

```diff
@@ -285,7 +283,7 @@ void Stop()  // - Hammertime!
 		g_video_backend->Video_ExitLoop();
 	}
 #if defined(__LIBUSB__) || defined(_WIN32)
```

Comment:
If we could limit it to one file, I'd prefer that. Especially since places like the UI need not really know whether we either have `libusb` or run on Windows. Perhaps at least replace it with a GC-Adapter specific define thats set in the header whenever those conditions are met, so we don't have to go all over the places if we suddenly have another way of this working without `libusb` on non-Windows.